src/components/UI/HomepageHero.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, Center, Stack, useMediaQuery } from '@chakra-ui/react';
+import Image from 'next/image';
+import { FC } from 'react';
+
+import { PageSubheading } from '../UI';
+import { ButtonLink } from '../';
+
+import homepageHero from '../../public/images/homepage-hero.svg';
+import homepageHeroMobile from '../../public/images/homepage-hero-mobile.svg';
+
+export const HomepageHero: FC = () => {
+  const [isMobile] = useMediaQuery('(max-width: 768px)');
+
+  return (
+    <Stack h='580px' mb={isMobile ? 6 : 16}>
+      <Stack px={5} py={3} mb={-10} zIndex={1}>
+        <PageSubheading
+          as='h1'
+          fontSize='homepage'
+          fontWeight={100}
+          lineHeight='36px'
+          textAlign='left'
+          mb={6}
+        >
+          We provide grants and other support to the builders of the Ethereum ecosystem.
+        </PageSubheading>
+
+        <Center>
+          <ButtonLink label='Learn more' link='#' width='177px' />
+        </Center>
+      </Stack>
+
+      <Box>
+        <Image
+          src={isMobile ? homepageHeroMobile : homepageHero}
+          alt='People gathered around the Ethereum tree'
```

Comment:
Nice a11y attention.

One thing (not for this PR) that you probably already know is that we should place this image differently, more as a background image, to support later the desktop layout.

Reply:
I'll make more research later on this. Maybe it's a bit tricker because the image in the design overlaps a box with a background color, not sure how to achieve the same result with a background image.

Reply:
Cool, got it. Yes, it looks a bit tricky.